backend/api/Models/Concepts/ResearchFile/PropertyModel.cs

```diff
@@ -0,0 +1,13 @@
+namespace Pims.Api.Models.Concepts
+{
+    // TODO: should this be refactored?
+    public class PropertyModel : BaseModel
```

Comment:
yeah, I'd assume that this would be a general property concept and not one specific to research (otherwise we are back to creating multiple models for every entity again). I'm doing this in my PR that is coming soon.

Reply:
That is what I thought too. For this PR I decided to leave most of the property model empty and only have what we need. In the future we should add to this model and move away from the other one.